### DISCUSSION TITLE
Add (automatic) expiry option

### DISCUSSION THREAD

**Discussion by snaeil** [lukevella/rallly#134] [Ideas]
## Current behaviour

No expiry option available.

## Requested behaviour

Like other tools (e.g. framadate) it would be awesome to

1. have a default automatic expiry date
2. to be able to set the expiry date within the poll creation

E.g. you could add an option, saying

> Your poll will automatically be deleted 180 days after the last date of your poll. You can set a specific expiry date for the poll:
> Expiry date: [DATE_PICKER]

**Comment by lukevella** [comment]
Moving to discussions
